stable/mpi-operator/templates/mpi-operator-clusterrole.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.rbac.clusterresources.create }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: mpi-operator
```

Comment:
all components should have the labels of app, heritage, release, chart (and maybe component). make sure to do this in podspec templates as well.